qcengine/programs/nwchem/harvester.py

```diff
@@ -729,6 +748,17 @@ def harvest(in_mol: Molecule, nwout: str, **kwargs) -> Tuple[PreservingDict, Non
     # Parse the NWChem output
     outPsivar, outMol, outGrad, version, error = harvest_output(nwout)
 
+    # If available, read higher-accuracy gradients
+    #  These were output using a Python Task in NWChem to read them out of the database
+    if outfiles.get("nwchem.grad") is not None:
+        logger.debug("Reading higher-accuracy gradients")
+        outGrad = json.loads(outfiles.get("nwchem.grad"))
```

Comment:
`out_grad` we try to stick to PEP8 variable names.

Reply:
Sure, I'll make the changes